nntrainer/tensor/tensor_base.h

```diff
@@ -568,7 +586,7 @@ class TensorBase {
    * @brief     Get size of current tensor
    * @retval    unsigned int size of the current tensor
    */
-  size_t size() const { return dim.getDataLen(); }
+  virtual size_t size() const { return dim.getDataLen(); }
```

Comment:
size() is now a virtual function since it will be overridden by a quantized tensor that utilizes data packing.